src/main/java/com/bisnode/opa/testformats/junit/TestCase.java

```diff
@@ -32,38 +32,38 @@ private TestCase(String name, String className, long time, boolean failure, Erro
     }
 
     @JacksonXmlProperty(isAttribute = true)
-    public String getName() {
```

Comment:
Are these not desirable outside of the package (removing the `public` modifier)?

Reply:
Nope, outside the package you only need `JUnitXml::asXmlString`. You shouldn't care about anything else.